.idea/codeStyleSettings.xml

```diff
@@ -16,8 +16,12 @@
           </value>
         </option>
         <JSCodeStyleSettings>
+          <option name="USE_SEMICOLON_AFTER_STATEMENT" value="false" />
+          <option name="FORCE_SEMICOLON_STYLE" value="true" />
           <option name="SPACE_BEFORE_FUNCTION_LEFT_PARENTH" value="false" />
           <option name="USE_DOUBLE_QUOTES" value="false" />
+          <option name="FORCE_QUOTE_STYlE" value="true" />
+          <option name="ENFORCE_TRAILING_COMMA" value="WhenMultiline" />
```

Comment:
Completely unrelated to this PR but noticed it while I was in our only js file...